asf_search/__init__.py

```diff
@@ -22,7 +22,7 @@
 from .ASFSession import ASFSession
 from .ASFProduct import ASFProduct
 from .ASFSearchResults import ASFSearchResults
-from .ASFSearchOptions import ASFSearchOptions, validators
+from .ASFSearchOptions import ASFSearchOptions, validators, validator_map
```

Comment:
This means you run `asf.validator_map` to access it. I can't make it `asf.ASFSearchOptions.validator_map`, because `asf.ASFSearchOptions` itself is a reference to the class, not the module. I don't *think* python lets you overload it?

Is it fine to keep it in the root of the package? Or should we move the class down one? It'd be a breaking change, so I'm less fond of that option, but maybe there's a middle ground I'm missing. It just seems weird having something no one will use, in the root of the package.